tests/edge-to-edge/server/app.spec.ts

```diff
@@ -297,8 +297,13 @@ describe( 'Termbox SSR', () => {
 			expect( $primaryLanguageFingerprint )
 				.toHaveAttribute( 'lang', language );
 
-			expect( $dom.querySelector( '.wikibase-termbox-fingerprint__language' ) )
+			const $language = $dom.querySelector( '.wikibase-termbox-fingerprint__language' ) as Element;
```

Comment:
```suggestion
			const $language = $dom.querySelector( '.wikibase-termbox-fingerprint__language' );
```

Reply:
Probably goes without saying but there were $elements which were further queried down the line, causing compiler errors because the previous query can yield null results and assertions do not act as sufficient guards as far as tsc is concerned.